manpages/awesome-client.1.de.txt

```diff
@@ -24,25 +24,25 @@ Wenn 'rlwrap' installiert ist wird es benutzt, um eine readline-Kommandozeilenei
 
 Das Modul 'awful.remote' muss geladen sein, um diesen Befehl zu benutzen.
 
-EXAMPLES
--------
+BEISPIELE
+---------
 ....
 $ awesome-client 'return 1+1, "Hello, world"\
    double 2
    string "Hello, world"
 
-# The following does not return anything on the command line,
-# but will show a notification.
+# Das folgende Beispiel erzeugt keine Ausgabe auf der Kommandozeile,
+# aber zeigt eine Benachrichtigung an.
```

Comment:
Besser "sondern zeigt eine"?

Reply:
Done (as a forced push) and thanks for taking a look!